x-pack/solutions/security/plugins/security_solution/public/siem_migrations/rules/pages/index.tsx

```diff
@@ -96,6 +91,9 @@ export const MigrationRulesPage: React.FC<MigrationRulesPageProps> = React.memo(
     const pageTitle = useMemo(() => <PageTitle />, []);
 
     const content = useMemo(() => {
+      if (ruleMigrationsStats.length === 0 && !migrationId) {
```

Comment:
Is the `!migrationId` condition necessary?

Reply:
Yes, otherwise it will display `Unknown` migration. I followed below guideline for showing `Empty` state.

It only shows up when:
- User visits `/rules` i.e. no `migrationId`.
- User does not have any migration created.

Am i missing something?